### DISCUSSION TITLE
Discord link invalid or expired.

### DISCUSSION THREAD

**Discussion by ringods** [lit/lit#4546] [General]
Hello,

When clicking the Discord link on the [Community](https://lit.dev/docs/resources/community/) page, I go through the CAPTCHA like process to end up with the message: `Discord link is invalid or expired`. Can someone verify this?

Ringo

**Comment by augustjk** [comment]
The links works for me. It's a redirect to https://discord.com/invite/buildWithLit which is still active as far as I can tell. I don't get any captcha either. Perhaps it's some issue with your network?

**Reply by ringods** [reply]
@augustjk I experienced the faulty procedure when executing it on my iPad. It worked on my laptop, but also with a captcha though.
